src/content/guides/using-persistent-volumes-with-csi-driver-on-aws/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Using Persistent Volumes with EBS CSI driver on AWS"
+description: Tutorial on how to use dynamically provisioned Persistent Volumes with EBS CSI driver on a cluster running on Amazon Web Services"
+type: page
+weight: 50
+tags: ["tutorial"]
+user_questions:
+  - How do I install the EBS CSI driver?
+  - How do I provision an EBS instance on AWS?
+  - How do I test the EBS CSI storage class?
+  - How do I use EBS CSI Volumes?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-firecracker
+---
+
+# Using Persistent Volumes on AWS with Container Storage Interface (CSI)
+
+CSI has been promoted to GA in Kubernetes v1.13 and is becoming the standard to replace the current Kubernetes `in-tree` storage plugin to handle Volumes for different provider.
+
+The `aws-ebs-csi-driver-app` provides a CSI interface to manage the lifecycle of Amazon EBS Volumes.
+
+## Installing EBS CSI driver
+
+To install the EBS CSI driver you will need to follow these steps:
+
+1. Access Giant Swarm web UI and select the cluster on which you want to install the EBS CSI driver.
+2. Open the _Giant Swarm Playground_ catalog.
+3. Write `aws-ebs-csi-driver-app` in the search bar.
+4. Click the _Configure & Install_ button, select the cluster and it will be installed.
```

Comment:
```suggestion
2. Open the Apps tab.
3. Click the Install App button
4. Select the Giant Swarm Playground catalog.
5. Select the App named aws-ebs-csi-driver-app.
6. Click the Configure & Install button. Make sure that the correct cluster is selected.
7. Click the Install App button.
```

Adding some intermediate steps. No need for using search here, as the app is in the first row.